tools/src/versioning/ios/transforms/postTransforms.ts

```diff
@@ -178,13 +178,23 @@ export function postTransforms(versionName: string): TransformPipeline {
         replace: /MessageHandlerName = @"ABI\d+_\d+_\d+ReactNativeWebView";/,
         with: `MessageHandlerName = @"ReactNativeWebView";`,
       },
+      {
+        paths: 'EXVersionManager.mm',
+        replace: /\[(RNCWebView)/,
+        with: `[${versionName}$1`,
+      },
 
       // react-native-reanimated
       {
         paths: 'EXVersionManager.mm',
         replace: /(_bridge_reanimated)/g,
         with: `${versionName}$1`,
       },
+      {
+        paths: 'EXVersionManager.mm',
+        replace: /\b(REA)/g,
+        with: `${versionName}$1`,
+      },
```

Comment:
assuming this is reanimated - is native proxy reanimated as well? if not can we make that clear

Reply:
Yes, it's from reanimated but its filename is not prefixed and it doesn't have to. It's C++ file so we only need to change the namespace of symbols in there 😉